### PR TITLE
Feature/fixnconf

### PR DESCRIPTION
**Pull request by faygao52**

### CODE REVIEW

File: lib/bilrost/bus.js [hunk 1]
```diff
@@ -25,9 +25,8 @@ module.exports = function() {
   /**
    * Setup nconf to use (in-order):
    * 1. Environment variables
-   * 2. A file located at './config.test.json'
+   * 2. Config file that the app is using
```
Comment (shirren) [inline lib/bilrost/bus.js]: @faygao52 Have we checked that the tests still pass? I think the tests rely on that line you have deleted.

File: lib/bilrost/worker.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ function Worker(topic, subscriber, options, callback, mockBus) {
   this.subscriber = subscriber;
   this.callback = callback;
   this.bus = mockBus || Bus;
+  this.receive = this.receive.bind(this);
```
Comment (shirren) [inline lib/bilrost/worker.js]: @faygao52 why was this line changed?

Comment (faygao52) [reply lib/bilrost/worker.js]: @shirren I don't know, I didn't change this

File: lib/bilrost/bus.js (hunk 1 above)
Comment (faygao52) [reply lib/bilrost/bus.js]: @shirren All the test passed
